src/common/benches/bench_hash_key_encoding.rs

```diff
@@ -50,10 +50,10 @@ impl HashKeyDispatcher for HashKeyBenchCaseBuilder {
         for null_ratio in NULL_RATIOS {
             for chunk_size in CHUNK_SIZES {
                 let id = format!(
-                    "{} {:?}, {} rows, Pr[null]={}",
+                    "{} rows, {} {:?}, Pr[null]={}",
```

Comment:
Prefix by rows so we can filter by it.
Criterion uses libtest as the runner. For libtest filtering, it only does substring match, regex not supported yet unfortunately..